web/gds-admin-ui/src/redux/autocomplete/saga.js

```diff
@@ -0,0 +1,20 @@
+const { call, put, takeEvery } = require("redux-saga/effects")
+const { default: getAllAutocomplete } = require("services/autocomplete")
+const { fetchAutocompleteApiResponseSuccess, FetchAutocompleteActionTypes, fetchAutocompleteApiResponseError } = require(".")
+
+function* fetchAutocompletes() {
```

Comment:
I haven't seen the `function*` keyword before, what does this do? Is it an ES2017 thing?

Reply:
It's an es6 thing, we call them `generators`